docs/tutorials/rootless_tutorial.md

```diff
@@ -0,0 +1,87 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+# Basic Setup and Use of Podman in a Rootless environment.
+
+Prior to allowing users without root privileges to run Podman, the administrator must install Podman and complete the following configurations.
+
+## Installing Podman
+
+For installing or building Podman, please see the [installation instructions](install.md).
+
+## Enable usernamespaces
+
+The number of usernamespaces that are allowed on the system is contained in the file `/proc/sys/user/max_user_namespaces`.  To set that to 15000, use the command:  `sysctl user.max_user_namespaces=15000`.
+
+## Install shadow-utils
+
+The `shadows-utils` package must be installed on the system.
+
+## /etc/subuid and /etc/subgid configuration
+
+Rootless podman requires the user running it to have a range of UIDs listed in /etc/subuid and /etc/subgid.
+
+Update the /etc/subuid and /etc/subgid with fields for users that look like:
+
+```
+cat /etc/subuid
+johndoe:100000:65536
+test:165536:65536
+```
+
+The format of this file is USERNAME:UID:RANGE
+
+* username as listed in /etc/passwd or getpwent.
+* The initial uid allocated for the user.
+* The size of the range of UIDs allocated for the user.
+
```

Comment:
Should add a NOTE stating that this is not currently supported with network installs. IE You have to have files like this locally, no way to set this up with LDAP or ActiveDirectory.

Reply:
Added note